sormas-backend/pom.xml

```diff
@@ -128,6 +128,10 @@
 			<groupId>fr.opensagres.xdocreport</groupId>
 			<artifactId>fr.opensagres.xdocreport.template.velocity</artifactId>
 		</dependency>
+        <dependency>
+            <groupId>fr.opensagres.xdocreport</groupId>
+            <artifactId>fr.opensagres.xdocreport.converter.docx.xwpf</artifactId>
+        </dependency>
```

Comment:
did you run the deploy serverlibs command?

Reply:
Yes, and it will be successful.
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/9607362d-b445-4518-aadd-d86adba826d3)

But in the sormas/lib the jar will not appear. Also tried to delete the manually added jar and run deploy serverlibs again but the jar will not appear and with the manually added jar the deployment continues to fail.